### ISSUE TITLE
Automatically starting MCSS via Task scheduler on Windows

### ISSUE THREAD

**Issue by Luporion** [mcserversoft/IssueTracker#17]
**Describe the bug**
mcss.exe only starts in background, no gui of the program is shown (also nothing in tray), but the webinterface works. When someone then starts the software manually, the gui shows up but actually another instance of MCSS is started

**To Reproduce**
1. Add a task starting mcss.exe **before** boot
2. log into OS
3. see no gui 🥲
4. start mcss thinking it would use the background process
5. it actually starts another instance with gui

**Expected behavior**
mcss should go to tray ot give some other way to access gui without closing the other instance and having to shut down all the servers

**Screenshots**
MCSS Webpanel working directly after boot:
![image](https://github.com/user-attachments/assets/6e9919ac-c9ea-472a-afb4-de78a45e8a9b)
MCSS in background after boot and login with MCSS not having UI:
![image](https://github.com/user-attachments/assets/19c18121-947b-4ec0-a7b5-5030c8a0d1a2)
MCSS not in tray or anywhere really:
![image](https://github.com/user-attachments/assets/9cb1b341-754a-41a4-8921-60d957cf84fd)
2nd MCSS instance when launching the program manually after login:
![image](https://github.com/user-attachments/assets/a3a2de54-431a-448d-a477-bd356888ca68)

**Desktop (please complete the following information):**
 - Windows 10 Server
 - Mcss Version 13.9.1

**Additional context**
I'm hosting a server for me and some friends. When im not home, i wake up the server using WOL (wake-on-lan) and a automatic task launches mcss.exe while another launches a VPN called tailscale (which i also have installed on my phone, so i would't have to expose the webpanel). I then utilize the webpanel to either manually start the server from my phones browser or a task whithin the webpanel automatically starts a server.
If there is any need to do anything else, i connect via RDP to look into the Desktop.

Am i missing something? Is there a way to configure this?

**Comment by Leonito2003** [comment]
Hi, first of all this issue isn't one and has nothing to do with mcss, it's just that you used "Run whether the user is logged in or not" in task scheduler, which by design runs tasks in the background with no gui whatsoever.

Second, you don't need to use task scheduler to start mcss, as the app already includes an option to start with windows, and even to start minimized in the tray if you want.

**Comment by Luporion** [comment]
No, Mcss only as a option to start at login. I'm talking about starting directly after boot, even before login.
Otherwise, someone would have to log in via RDP or autologin needs to be enabled, which is horrendously insecure.

**Comment by Leonito2003** [comment]
After a quick search, it seems like you can configure windows to open the session but keep it locked at boot, however your need the pro edition as this is a group policy : https://superuser.com/a/1786930/1067313
